Twig/AsseticExtension.php

```diff
@@ -21,7 +21,7 @@
  *
  * @author Kris Wallsmith <kris@symfony.com>
  */
-class AsseticExtension extends BaseAsseticExtension
+class AsseticExtension extends BaseAsseticExtension implements \Twig_Extension_GlobalsInterface
```

Comment:
Unfortunately the build fails because of this deprecation...
This will be fixed when https://github.com/kriswallsmith/assetic will have a new release.